fares-data-build-tool/fdbt-site
# feat: Multi operator TXC switch enabled

# Description

-   To enable the user to switch the TXC data source from TNDS to BODS (and back) for the other operators as part of the multiple operator journey.

# Testing instructions

-   Run site locally and use multi operator multi service journey. Also use the serviceList afterwards / before for the flat fare journey to ensure no regressions across that.

# Type of change


# Checklist:
